pkg/client/volume/volume.go

```diff
@@ -54,13 +52,7 @@ func (c *Client) DeleteVolume(id string) error {
 }
 
 func (c *Client) GetAll() (*models.Volumes, error) {
-	klog.V(1).Info("Calling the Power Volumes GetAll Method")
-	params := p_cloud_volumes.NewPcloudCloudinstancesVolumesGetallParamsWithTimeout(pkg.TIMEOUT).WithCloudInstanceID(c.instanceID)
-	resp, err := c.session.Power.PCloudVolumes.PcloudCloudinstancesVolumesGetall(params, c.session.AuthInfo(c.instanceID))
-	if err != nil {
-		return nil, errors.ToError(err)
-	}
-	return resp.Payload, nil
+	return c.client.GetAll()
```

Comment:
A definition for GetAll() exists in power-go-client. The same has been re-used as the functionality is indifferent from the current implementation. 
```
	params := p_cloud_volumes.NewPcloudCloudinstancesVolumesGetallParams().
		WithContext(f.ctx).WithTimeout(helpers.PIGetTimeOut).
		WithCloudInstanceID(f.cloudInstanceID)
	resp, err := f.session.Power.PCloudVolumes.PcloudCloudinstancesVolumesGetall(params, f.session.AuthInfo(f.cloudInstanceID))
	if err != nil {
		return nil, ibmpisession.SDKFailWithAPIError(err, fmt.Errorf("failed to Get all Volumes for Cloud Instance %s: %w", f.cloudInstanceID, err))
	}
	if resp == nil || resp.Payload == nil {
		return nil, fmt.Errorf("failed to Get all Volumes for Cloud Instance %s", f.cloudInstanceID)
	}
	return resp.Payload, nil
```